src/js/Controllers/UIController.js

```diff
@@ -291,6 +291,18 @@ class UIController {
         button.mode = "SHORT"
         this.listener.send(RpcFactory.OnButtonPressNotification(appID, button))
     }
+    onLongButtonPress(appID, buttonID, buttonName) {
+        var button = {
+            name: buttonName,
+            mode: "BUTTONDOWN",
+            customButtonID: buttonID
+        }
```

Comment:
ButtonEvent messages should be sent in their respective functions, not all at once